docs/development/releasing.rst

```diff
@@ -148,20 +153,11 @@ packages that use the full bugfix/maintenance branch approach.)
 
       $ conda create -n astropy_release_test_v<version> numpy
       $ source activate astropy_release_test_v<version>
+      $ pip install .[all]
```

Comment:
this would also install astropy, from the local directory, doesn't it?

Reply:
No idea. I haven't use `pip` like that much yet. Should I add a `pip uninstall astropy` after? Or should I find another way to "install all the dependencies" like @astrofrog wanted in #5120 ?

Reply:
A cleaner approach is to combine this with the line below:

```
pip install dist/astropy-<version>.tar.gz[all]
```